doc/man7/provider-base.pod

```diff
@@ -620,46 +620,58 @@ to the list of signature algorithms that libssl already supports.
 
 Each TLS signature algorithm that a provider supports should be described via
 the callback passed in through the provider_get_capabilities function. Each
-algorithm should have the following details supplied (all are mandatory):
+algorithm can have the following details supplied:
 
 =over 4
 
 =item "tls-sigalg-name" (B<OSSL_CAPABILITY_TLS_SIGALG_NAME>) <UTF8 string>
 
 The name of the signature algorithm as given in the IANA TLS Signature Scheme registry
 L<https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#tls-signaturescheme>.
+This value is mandatory to be supplied.
 
 =item "tls-sigalg-name-internal" (B<OSSL_CAPABILITY_TLS_SIGALG_NAME_INTERNAL>) <UTF8 string>
 
 The name of the signature algorithm as known by the provider. This could be
 the same as the "tls-sigalg-name", but does not have to be.
+This value is mandatory to be supplied.
 
 =item "tls-sigalg-code-point" (B<OSSL_CAPABILITY_TLS_SIGALG_CODE_POINT>) <unsigned integer>
 
 The TLS algorithm ID value as given in the IANA TLS SignatureScheme registry.
+This value is mandatory to be supplied.
 
 =item "tls-sigalg-alg" (B<OSSL_CAPABILITY_TLS_SIGALG_ALG>) <UTF8 string>
 
 The name of a Key Management algorithm that the provider offers and that should
 be used with this algorithm.
 The algorithm must support key and parameter generation as well as sign and
 verify operations.
+This value is mandatory to be supplied.
 
 =item "tls-sigalg-oid" (B<OSSL_CAPABILITY_TLS_SIGALG_OID>) <UTF8 string>
 
 The OID of the signature algorithm.
+This value is optional to be supplied. The OID must be registered
+via core_obj_create().
```

Comment:
Well, in theory I suppose it could already be in our OID database. Perhaps make this "The OID must be registered, e.g. via core_obj_create()"

Reply:
See updated text in https://github.com/openssl/openssl/pull/19312/commits/8be6985a9743a239684308b6258691d3489e52bf. This refers to the logic in https://github.com/baentsch/openssl/blob/949590872922dd9dda6c73d216f3ea748b3bb3ce/ssl/t1_lib.c#L583-L594